src/scss/colorways.scss

```diff
@@ -390,6 +403,19 @@
   color: #22aabc;
 }
 
+.gmk-deku-key {
+  background: #298286;
+  color: #e5e1de;
```

Comment:
can we be consistent? pick either the rgb form or the # form. don't do both.

Reply:
> can we be consistent? pick either the rgb form or the # form. don't do both.

it's ok now since I changed GMK Camping to SP color code